app/src/main/res/layout/recent_bookmarks_header.xml

```diff
@@ -12,31 +12,34 @@
     android:layout_marginTop="40dp">
 
     <androidx.appcompat.widget.AppCompatTextView
-        android:id="@+id/recentlySavedBookmarksHeader"
+        android:id="@+id/header"
         style="@style/Header20TextStyle"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:contentDescription="@string/recently_saved_bookmarks_content_description"
-        android:maxLines="1"
-        android:paddingTop="16dp"
-        android:paddingBottom="16dp"
+        android:maxLines="2"
+        android:paddingVertical="16dp"
         android:text="@string/recently_bookmarked"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
     <com.google.android.material.button.MaterialButton
         android:id="@+id/showAllBookmarksButton"
-        style="@style/Button12TextStyle"
+        style="@style/Button15TextStyle"
```

Comment:
I think you are looking at the iOS designs. The buttons for Android should be 14dp and purple in light theme.

Reply:
Duh. I was wondering why it was blue. Thanks for the catch 👍